tests/unit/web/dosguard/WhitelistHandlerTests.cpp

```diff
@@ -47,7 +47,7 @@ getParseWhitelistHandlerConfig(boost::json::value val)
     ConfigFileJson const jsonVal{val.as_object()};
     auto config = ClioConfigDefinition{{"dos_guard.whitelist.[]", Array{ConfigValue{ConfigType::String}}}};
     auto const errors = config.parse(jsonVal);
-    ASSERT(!errors.has_value(), "Cannot parse Json Correctly");
+    [&]() { ASSERT_FALSE(errors.has_value()); }();
```

Comment:
fyi: if you want to preserve the custom error message you can still do it this way:
```cpp
ASSERT_FALSE(condition) << "Custom error message";
```